sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -21,3 +23,16 @@ export class AssessmentHistorySummaryAPIOutDTO {
   studentAppealId?: number;
   studentScholasticStandingId?: number;
 }
+
+export class AssessmentNOAAPIOutDTO {
+  assessment: Assessment;
```

Comment:
we can have a todo comment here to create a copy of Assessment as DTO object with DTO naming convention. I believe it is currently an interface and hence it may not appear properly in swagger documentation.

Reply:
Assessment is dynamic and we mapped in the API to be more explicit while looking into the code. Converting to a class at the DTO level will force the removal of possible properties that would not be mapped to the DTO, right?